indico/modules/events/editing/client/js/management/editable_type/EditableTypeDashboard.jsx

```diff
@@ -93,9 +93,9 @@ export default function EditableTypeDashboard() {
               }
               description={Translate.string('Toggle whether users can submit new editables')}
             >
-              <a className="i-button highlight" onClick={toggleSubmission}>
+              <button className="i-button highlight" onClick={toggleSubmission} type="button">
```

Comment:
I am receiving here a warning that buttons in `EditableTypeDashboard.jsx` file need a type. Is the `button` type right for these elements?